docker-compose.yml

```diff
@@ -156,13 +156,14 @@ services:
       - db
       - prometheus
     environment:
-      - DATABASE_SERVICE_NAME=postgres
-      - POSTGRES_SERVICE_HOST=db
-      - POSTGRESQL_DATABASE=compliance_dev
-      - POSTGRESQL_TEST_DATABASE=compliance_test
-      - POSTGRESQL_USER=insights
-      - POSTGRESQL_PASSWORD=insights
-      - SETTINGS__PROMETHEUS_EXPORTER_HOST=prometheus
+#      - DATABASE_SERVICE_NAME=postgres
+#      - POSTGRES_SERVICE_HOST=db
+#      - POSTGRESQL_DATABASE=compliance_dev
```

Comment:
Sorry, still using this compose config for testing other work, and I found an issue. This compliance_dev DB name is removed completely and not used within test.json. As you can see, some services need the insights DB (i.e inventory-web and inventory) while all of our compliance services need compliance_dev DB.

Reply:
Similar comment for the tests which use POSTGRESQL_TEST_DATABASE which is different than the devel DB.

Reply:
okay, I updated the Clowder settings mock file to provide the "compliance_dev" database name, left the `POSTGRESQL_TEST_DATABASE` as fixed environment variable - since this is only for local deployment I think it should be okay.